x/profiles/module_ibc.go

```diff
@@ -179,8 +279,28 @@ func (am AppModule) OnAcknowledgementPacket(
 // OnTimeoutPacket implements the IBCModule interface
 func (am AppModule) OnTimeoutPacket(
 	ctx sdk.Context,
-	modulePacket channeltypes.Packet,
+	packet channeltypes.Packet,
 ) (*sdk.Result, error) {
+	var data oracletypes.OracleRequestPacketData
+	if err := types.AminoCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
```

Comment:
```suggestion
	if err := types.ProtoCdc.UnmarshalJSON(packet.GetData(), &data); err != nil {
```
The codec for packet should be `ProtoCdc`.